dev-requirements.txt

```diff
@@ -1,58 +1,109 @@
 -i https://pypi.python.org/simple
+-e .
+-e git+https://github.com/nucypher/py-solc.git@391b8da1a6bac5816877197bda25527c6b0b8c15#egg=py-solc
 alabaster==0.7.12
 ansible==2.7.7
 apipkg==1.5
+appdirs==1.4.3
+argh==0.26.2
 asn1crypto==0.24.0
 atomicwrites==1.3.0
+attrdict==2.0.1
 attrs==18.2.0
+autobahn==19.2.1
+automat==0.7.0
 aws-xray-sdk==0.95
 babel==2.6.0
 bcrypt==3.1.6
-boto3==1.9.91
+boto3==1.9.96
 boto==2.49.0
-botocore==1.12.91
+botocore==1.12.96
 bumpversion==0.5.3
+bytestring-splitter==1.0.0a4
 certifi==2018.11.29
-cffi==1.11.5
+cffi==1.12.0
 chardet==3.0.4
+click==7.0
+coincurve==11.0.0
+colorama==0.4.1
 commonmark==0.8.1
+constant-sorrow==0.1.0a8
+constantly==15.1.0
 coverage==4.0.3
 cryptography==2.5
+cytoolz==0.9.0.1 ; implementation_name == 'cpython'
+dateparser==0.7.1
 decorator==4.3.2
 docker-pycreds==0.4.0
 docker==3.7.0
 docutils==0.14
 ecdsa==0.13
+eth-abi==2.0.0b5
+eth-account==0.3.0
+eth-bloom==1.0.3
+eth-hash[pycryptodome,pysha3]==0.2.0
+eth-keyfile==0.5.1
+eth-keys==0.2.1
+eth-rlp==0.1.2
+eth-tester==0.1.0b37
+eth-typing==2.0.0
+eth-utils==1.4.1
+ethpm==0.1.4a12
```

Comment:
This package depends on `ethereum/py-solc` from PYPI; which conflicts with `nucypher/py-solc` (which I also do not see here in `dev-requirementst.txt` - see `master` for an example) - so merging this PR into master may result in build failure.

Reply:
It was prepared using `pipenv`
How properly to fix this?
Remove `ethpm`?
Or take file from `master`?

Reply:
Perhaps, for now, we may fork py-solc (ugh!) and put it into a different namespace.
If we're uncomfortable with putting it in pypi, we can put that right into nucypher repository (I know, that is not nice)

Reply:
@KPrasch and I actually had a conversation about it in Denver. It's a bit trickier if we need both.
A little bit easier if we need only `nucypher/py-solc`

Reply:
I have needed to manually change requirements files generated by pipenv.  I can successfully run the project by removing `eth-pm`, then installing `nucypher/py-solc` from GH.  installing `web3` will result in the installation of `eth-pm` (and thus `ethereum/py-solc`) so it must be done in this order.

Reply:
Perhaps, just take the requirements files from `master` and update them manually.